pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -47,7 +48,12 @@ func (s *AWSCloud) ensureLoadBalancer(name string, listeners []*elb.Listener, su
 
 		createRequest.SecurityGroups = stringPointerArray(securityGroupIDs)
 
-		glog.Info("Creating load balancer with name: ", name)
+		createRequest.Tags = []*elb.Tag{
+			{Key: aws.String(TagNameKubernetesCluster), Value: aws.String(s.getClusterName())},
```

Comment:
Ah ... good catch.  Including the cluster tag is a good thing to do.  ELB isn't quite as good at letting us actually use these tags easily, but we should still tag it.